arroyo-types/Cargo.toml

```diff
@@ -5,4 +5,7 @@ edition = "2021"
 
 [dependencies]
 bincode = "2.0.0-rc.3"
+base64 = "0.21"
+md-5 = "0.10"
+sha2 = "0.10"
```

Comment:
Usually I leave these unpinned on the minor version, in order to stay up to date with patches, but I've seen it both ways.  What do we prefer here?

Reply:
Yeah, that's fine.